src/main/java/it/pagopa/transactions/controllers/v2/TransactionsController.java

```diff
@@ -58,6 +63,12 @@ public Mono<ResponseEntity<ProblemJsonDto>> openStateHandler() {
                                 .detail("Upstream service temporary unavailable. Open circuit breaker."),
                         HttpStatus.BAD_GATEWAY
                 )
+        ).doOnNext(
+                ignored -> openTelemetryUtils.addErrorSpanWithAttributes(
```

Comment:
Any specific reasons for not using `OpenTelemetryUtils#addErrorSpanWithException`?

Reply:
Yes, I missed it :) I update it

Reply:
Fixed by https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/503/commits/00ef6daff349196c50e68975f0a3f2415fb24cbd